packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -117,12 +119,16 @@ ${Object.keys(authorsMap)
       ...getAuthorsMapAuthor(frontMatterAuthor.key),
       ...frontMatterAuthor,
     };
+    const normalizedSocials = normalizeSocials(frontMatterAuthor.socials ?? {});
```

Comment:
This works but doesn't look like the best place to add this logic.

I'd add it there instead:

![CleanShot 2024-08-20 at 12 15 02@2x](https://github.com/user-attachments/assets/edba14ed-8fbe-4e7f-9de1-fb735463cf3a)